build-system/pr-check/remote-tests.js

```diff
@@ -49,7 +49,10 @@ async function main() {
 
     await startSauceConnect(FILENAME);
     timedExecOrDie('gulp unit --nobuild --saucelabs');
-    timedExecOrDie('gulp integration --nobuild --compiled --saucelabs');
+    timedExecOrDie(
+      'gulp integration --nobuild --compiled --saucelabs --stable'
+    );
+    timedExec('gulp integration --nobuild --compiled --saucelabs --beta');
```

Comment:
Now that the `zero tests detected by Karma` error doesn't exit the process, I'm trying to think why we have `timedExec` here instead of `timedExecOrDie`

Reply:
Good point. Restored to `timedExecOrDie`.